CONTRIBUTING.md

```diff
@@ -309,6 +309,25 @@ about:
     when someone else is asking for feedback on their code, and universe balances out.
 
 
+Backporting
+-----------
+
+After the original PR was merged into master, backporting is done only
+when necessary - the fix either needs to be present in a release branch
+(e.g. a security issue) or on request.
+If the backport is trivial and can be cherry-picked automatically,
+just request a "Needs backport" tag for the original PR.
+If backport needs any conflict resolution, create the PR manually and
+wait for comments.
+
+The commit message of a backport should additionally contain following
```

Comment:
`Each commit message of a backport ...`

Reply:
Thanks. Fixed in 163fe28c54105fdde687cc49c51e635d6e483ccb and will be squashed later into the final commit.